### PR TITLE
Overhaul W1D1T2

### PR DESCRIPTION
**Pull request by patrickmineault**
* Fix breaking bug in definition of loss (last time step vs. whole time period)
* Stage model in two steps: first unregularized, then regularized
* Change names: Simple vs Complex -> Regularized vs Unregularized
* Fix train/val folds across regularized vs unregularized
* Remove test fold
* Use threshold training loss to end training
* Use data without multiple delays during robustness testing, 8x faster
* Increase batch size to 20, much faster training
* Add text, equations, figures
* Streamline exercises
* Add plotting predicted muscle activity
* Add loading and presentation of spike data
* Change the method of calculating the normalization during robustness testing to something less controversial
* Remove the third regularization loss in the regularized RNN (too complicated to explain, low educational value)

The robustness results have been fixed.

### CODE REVIEW

**Comment by iamzoltan** [comment]
I will merge your branch into a branch called release-v0.0.1 so that the check will run

**Review by iamzoltan — COMMENTED** [review]
Looks great! Did you restart the kernel and clear all output before saving? I see a lot of widget state.

**Comment by patrickmineault** [comment]
I believe so, yes. I cleared the outputs as you can see from the rendered notebook.

**Comment by iamzoltan** [comment]
Any idea why there is many lines of widget state saved?
